### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by wlach**
More or less just a bunch of random stuff that needed to be fixed + some annotations to make errors easier to track down in the future.

### CODE REVIEW

File: missioncontrol/etl/measuresummary.py [hunk 1]
```diff
@@ -17,15 +17,19 @@ def _get_summary_dict(values, version=None):
...
-            "mean": None,
-            "usageHours": 0
+            "median": None,
+            "stdev": 0.0,
+            "usageHours": 0.0
         }
 
     normalized_values = [v[0]/(v[1]/1000.0) for v in values]
+    print(normalized_values)
+    print(values)
```
Comment (robhudson) [inline missioncontrol/etl/measuresummary.py]: Are these prints supposed to still be here?

File: tests/test_measuresummary.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        build_id='20170629075044',
+        version='55.0')
+    measure = Measure.objects.get(name=measure_name,
+                                  platform__name=platform_name)
+    series = Series.objects.create(build=build, measure=measure)
+    latest_timestamp = None
+    for i in range(num_datapoints):
+        latest_timestamp = base_datapoint_time + datetime.timedelta(hours=i)
+        (value, usage_hours, client_count) = (float(i), float(i + 1),
+                                              i)
```
Comment (robhudson) [inline tests/test_measuresummary.py]: Seems like an unnecessary linebreak. (Also LHS parens not needed, but that's more of a personal choice.)

File: tests/test_api_experiment.py [hunk 3]
```diff
@@ -22,3 +23,15 @@ def test_experiments_api(fake_experiments_data, client):
                 ['2017-07-01T12:00:00Z', 10.0, 20.0]
         ]
     }
+
+
+@pytest.mark.parametrize('missing_param', ['experiment', 'measure', 'interval'])
```
Comment (robhudson) [inline tests/test_api_experiment.py]: The `parametrize` helper is pretty cool 👍

**Comment by robhudson** [comment]
Made a few comments. Looks good. r+wc

**Comment by wlach** [comment]
Thanks for the nits, I will fix all of them before landing :)

**Pull request merged**